acct.bionimbus.org/portal/gitops.json

```diff
@@ -44,32 +44,46 @@
       },
       "buttons": [
         {
-          "name": "Data Model",
-          "icon": "data-field-define",
-          "body": "The ACCOuNT Data Commons ingests and exports data that conform to a graph-like relational data model.",
+          "name": "Define Data Field",
+          "icon": "acct-planning",
+          "body": "The ACCOuNT Data Commons ingests and exports data that conform to a graph-like relational data model. Study the dictionary to send GraphQL queries or format your data submissions.",
           "link": "/DD",
-          "label": "Learn more"
+          "label": "Learn more",
+          "color": "#1a3057"
         },
         {
           "name": "Explore Data",
-          "icon": "data-explore",
-          "body": "The File Exploration Page supports browsing data files using filters and selection of files for download.",
-          "link": "/files",
-          "label": "Explore data"
+          "icon": "acct-explore",
+          "body": "Use the Exploration tool for cohort building via faceted search and downloading subsets of data from the ACCOuNT Data Commons.",
+          "link": "/explorer",
+          "label": "Explore data",
+          "color": "#1a3057"
         },
         {
           "name": "Access Data",
-          "icon": "data-access",
-          "body": "Subject and file metadata can be queried using graphQL.",
+          "icon": "acct-access",
+          "body": "Subject and file metadata can be queried using GraphQL.",
           "link": "/query",
-          "label": "Query data"
+          "label": "Query data",
+          "color": "#1a3057"
         },
         {
           "name": "Analyze Data",
-          "icon": "data-analyze",
+          "icon": "acct-analyze",
           "body": "Analyze your selected cases using Jupyter Notebooks in our secure cloud environment",
-          "link": "/workspace",
-          "label": "Run analysis"
+          "link": "#hostname#workspace/",
```

Comment:
no

Reply:
This should be `"link": "/workspace",`.
We should fix this in `gitops-qa/qa-acct` to fix the coinfig replication for subsequent release cycles.